src/clp_ffi_js/ir/StreamReader.hpp

```diff
@@ -121,11 +121,11 @@ class StreamReader {
      * @param begin_idx
      * @param end_idx
      * @param use_filter Whether to decode from the filtered or unfiltered log events collection.
-     * @return An array where each element is a decoded log event represented by an array of:
-     * - The log event's message
-     * - The log event's timestamp as milliseconds since the Unix epoch
-     * - The log event's log level as an integer that indexes into `cLogLevelNames`
-     * - The log event's number (1-indexed) in the stream
+     * @return An object representing the decoded log events. Each decoded event includes:
```

Comment:
https://github.com/y-scope/clp-ffi-js/pull/98#discussion_r2164678262
copying from above since the new comment was made from outdated code

@hoophalab to be precise, instead of returning an array of arrays, now we're returning an array of objects, so i do think it's worth mentioning "array" in the return docstring.

to make this more clear, maybe we can improve it as:
```
@return An array of objects, where each object represents a decoded log event with the following properties:
...
```